MAINTAINERS

```diff
@@ -96,7 +96,9 @@ S: Maintained
 F: xlators/features/changelog/
 
 Distributed Hashing Table (DHT)
-P: Susant Palai <spalai@redhat.com>
+M: Barak Sason Rofman <bsasonro@redhat.com>
```

Comment:
Should we add @mohit84  too? Or he wouldn't be interested in this? I remember he looked after component for sometime.

Reply:
I would be interested, Please add me also.

Reply:
Done